blog/2015/06/training-lessons.html

```diff
@@ -0,0 +1,189 @@
+---
+layout: blog
+root: ../../..
+author: Greg Wilson
+title: "Training Lessons"
+date: 2015-06-22
+time: "13:00:00"
+category: ["Instructor Training"]
+---
+<!-- start excerpt -->
+<p>
+  I wrote about
+  <a href="{{page.root}}/blog/2015/05/online-instructor-training.html">our experiments with the format of instructor training</a>
+  back in May.
+  At that time,
+  we had run the class in three formats:
+</p>
+<ul>
+  <li>a multi-week online class,</li>
+  <li>an in-person two- or three-day class, and</li>
+  <li>the trainees physically together for two days with the trainer coming in via teleconference.</li>
+</ul>
+<p>
+  We have since tried it twice with the trainees at three different sites
```

Comment:
I think you need to clarify this sentence by noting somehow that "it" is the teleconference method. The sentence is ambiguous as it currently stands. You could also use the word "simultaneously" in here somewhere so that it's clear what you mean by "twice at three sites".